src/Cell.cpp

```diff
@@ -1313,15 +1313,23 @@ void Cell::sectorize(std::vector<Cell*>& subcells) {
     log_printf(DEBUG, "Creating a new sector Cell %d for Cell %d",
                sector->getId(), _id);
 
-    /* Add new bounding planar Surfaces to the clone */
-    sector->addSurface(+1, planes.at(i));
-
     if (_num_sectors != 2) {
+      /* Add new bounding planar Surfaces to the clone */
+      sector->addSurface(+1, planes.at(i));
+      
       if (i+1 < _num_sectors)
         sector->addSurface(-1, planes.at(i+1));
       else
         sector->addSurface(-1, planes.at(0));
     }
+    else {
+      /* for _num_sectors==2, planes[0] and planes[1] are actually the same but 
+         opposite direction, so the two adjacent sectors will have the same 
+         Halfspace value, which will cause trouble when a point is on the plane.
+         This is to avoid it. */
```

Comment:
For _num...
This is meant to avoid it OR This is a workaround

Reply:
This is to avoid this trouble.